frontend/src/app/testResults/TestResultPrintModal.test.tsx

```diff
@@ -77,7 +79,95 @@ describe("TestResultPrintModal", () => {
     expect(screen.getByText("01/28/2022 5:56pm")).toBeInTheDocument();
   });
 
+  it("should render only COVID information", () => {
+    expect(screen.getByText("Test result: COVID-19")).toBeInTheDocument();
+  });
+
   it("matches screenshot", () => {
     expect(component).toMatchSnapshot();
   });
 });
+
+if (process.env.MULTIPLEX_ENABLED === "true") {
+  describe("TestResultPrintModal with multiplex results in SimpleReport App", () => {
+    let component: any;
+
+    beforeEach(() => {
+      const multiplexTestResult = cloneDeep(testResult);
+      multiplexTestResult.results = [
+        { disease: { name: "COVID-19" }, testResult: "POSITIVE" },
+        { disease: { name: "Flu A" }, testResult: "POSITIVE" },
+        { disease: { name: "Flu B" }, testResult: "POSITIVE" },
+      ];
+
+      ReactDOM.createPortal = jest.fn((element, _node) => {
+        return element;
+      }) as any;
+
+      MockDate.set("2021/01/01");
+      component = render(
+        <DetachedTestResultPrintModal
+          data={{ testResult: multiplexTestResult }}
+          testResultId="id"
+          closeModal={() => {}}
+        />
+      );
+    });
+
+    it("should render flu information", () => {
+      expect(
+        screen.getByText("Test results: COVID-19 and flu")
+      ).toBeInTheDocument();
+      expect(screen.getByText("For flu A and B:")).toBeInTheDocument();
+    });
+
+    it("matches screenshot", () => {
+      expect(component).toMatchSnapshot();
+    });
+  });
+
+  describe("TestResultPrintModal with multiplex results in Pxp App", () => {
+    let component: any;
+
+    beforeEach(() => {
+      const multiplexPxpTestResult = cloneDeep(testResult);
+      multiplexPxpTestResult.results = [
+        // @ts-ignore
```

Comment:
⚠️ It didn't seem like there was a nice way to supress typescript warnings in a block
Added this test to check that the print page does work for patient experience side

Reply:
@emyl3 the best way to avoid this `ts-ignore` is to type the underlying variable, in this case `multiplexPxpTestResult`. 

Playing around with it, if you do something like:
`const multiplexPxpTestResult: VerifyV2Response = cloneDeep(testResult)`
Then IntelliJ should throw a prompt that the result is missing some fields from VerifyV2Response and suggest an alternate type. You can then pull that generated type up to the top of the file and use it in both tests (though you'll hit the `testResult`/`result` issue again and need to manually edit the generated type a little bit.)

It's best practice to type all your variables in TypeScript, if possible - it prevents you from hitting some of the unexpected behavior that comes with dynamic typing.